tests/functional/i/import_error.rc

```diff
@@ -1,5 +1,9 @@
 [Messages Control]
 disable=C,R,W
+enable=multiple-imports
```

Comment:
Maybe we should create a specific file for this one as it affect the whole pre-existing file.

Reply:
I just added this because the original test also checked this message. I think this was mostly a by-product of the actual thing it was trying to test. because all messages are enabled I needed to re-enable it. There are actual tests for `multiple-imports`. I can either remove this and the test will pass without showing that this message is also raised or I can leave as is. Moving to other file doesn't make a lot of sense since the actual message that is being tested is `import-error`.

Reply:
Ok, if you thought about it let's keep it that way !